Casamari/MNC010.xml

```diff
@@ -122,77 +215,163 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                              Rough sketch of a <foreign xml:lang="gez"> ḥarag</foreign> in pen on
+                                 <locus target="#1v #41r"/>.</desc>
+                        </item>
+
+                        <item xml:id="e4">
+                           <desc> A sticker of the monastic library with the signature ‘Ms 38’
+                              printed on it is glued onto the inner side of the front board. The
+                              signature Et III is written in pencil <locus target="#1r"/>.</desc>
+                        </item>
+                     </list>
                   </additions>
-            
+
                   <bindingDesc>
-                     <binding contemporary="true">
-                        <decoNote xml:id="b1">Two wooden boards covered with reddish-brown tooled leather. Three pairs of sewing stations. Textile inlays.
-                        </decoNote>
+                     <!-- TO BE CHECKED BY MARCO-->
+                     <binding contemporary="false" xml:id="binding">
+                        <decoNote xml:id="b1">Two<material key="wood">wooden</material> boards
+                           covered with tooled dark reddish-brown <material key="leather"/>, textile
+                           inlays.</decoNote>
                         <decoNote xml:id="b2" type="bindingMaterial">
                            <material key="wood"/>
+                           <material key="leather"/>
+                           <material key="textile"/>
                         </decoNote>
-                        <decoNote xml:id="b3" type="Endbands"/>
-                        <decoNote xml:id="b4" type="Other"/>
-                        <decoNote xml:id="b5">Blind tooled ornament on the cover, and on the edges.</decoNote>
+                        <decoNote xml:id="b3" type="Cover" color="brown">Skilled blind-tooling
+                           decoration on the boards: two frames with a cross inside, the turn-ins
+                           and the edges are carfully tooled with lines forming crosses and double
+                           circles.</decoNote>
+                        <decoNote xml:id="b4" type="SewingStations">6 (The original binding showed
+                           only one sewing station. The Ms received six sewing stations during the
+                           resoration process (<ref target="#condition"/>).</decoNote>
                      </binding>
                   </bindingDesc>
                </physDesc>
+
                <history>
                   <origin>
-                     <origDate notBefore="1700" notAfter="1899"/>
-                     <note/>
-                     <p/>
-                  </origin>
+                     <origDate notBefore="1700" notAfter="1899" evidence="lettering"/> The earliest
+                     date of production – taken the handwriting into account – can be set to the
```

Comment:
quoting Jacopo from the other PR: "based on the handwriting" or "taking into account the handwriting""